packages/puppeteer-core/src/cdp/ExecutionContext.ts

```diff
@@ -87,6 +96,14 @@ export class ExecutionContext implements Disposable {
       'Runtime.bindingCalled',
       this.#onBindingCalled.bind(this)
     );
+    this.#clientEmitter.on('Runtime.executionContextDestroyed', async event => {
```

Comment:
Now that I take a look `this.#clientEmitter` can be local variable in the constructor.